autoload/neomake/makers/ft/cs.vim

```diff
@@ -10,3 +10,12 @@ function! neomake#makers#ft#cs#mcs()
         \ 'errorformat': '%f(%l\,%c): %trror %m',
         \ }
 endfunction
+
+function! neomake#makers#ft#cs#msbuild()
+    return {
+        \ 'exe': 'msbuild',
+        \ 'args': [ '-nologo', '-v:q', '-property:GenerateFullPaths=true', neomake#utils#FindGlobFile(expand('%:p:h'), '*.sln') ],
+        \ 'errorformat': '\ %#%f(%l\\\,%c):\ %m',
```

Comment:
No need to escape the space here, i.e. replace `\ ` with just ` `.
Not sure about the `\\\`?!